src/rdf4cpp/BlankNode.cpp

```diff
@@ -56,6 +56,11 @@ BlankNode BlankNode::find(std::string_view identifier, storage::DynNodeStoragePt
 std::string_view BlankNode::identifier() const noexcept { return handle_.bnode_backend().identifier; }
 
 bool BlankNode::serialize(writer::BufWriterParts const writer) const noexcept {
+    if (null()) {
```

Comment:
I think the version from `Node` is better (i.e. `return writer::write_str("null", writer)`)